exp/views/study.py

```diff
@@ -89,10 +93,6 @@ def get_queryset(self, *args, **kwargs):
                 # TODO optimize using subquery
                 queryset = sorted(queryset, key=lambda t: t.end_date or timezone.now(), reverse=True if '-' in sort else False)
```

Comment:
Putting aside for now, follow-up ticket in JIRA created https://openscience.atlassian.net/browse/LEI-434